libraries/src/Component/Router/Rules/PreprocessRules.php

```diff
@@ -133,7 +134,7 @@ public function preprocess(&$query)
 
         // If we have a parent key and it is missing, lets add it
         if ($parent_key && $this->parent_key && !isset($query[$parent_key])) {
-            $query[$parent_key] = $obj->{$this->parent_key};
+            $query[$parent_key] = (string) $obj->{$this->parent_key};
```

Comment:
```suggestion
            $query[$parent_key] = $obj->{$this->parent_key};
```

I'm also reverting this for now.